datafusion/sqllogictest/test_files/timestamps.slt

```diff
@@ -1922,6 +1951,65 @@ true true true true true true
 #----
 #0001-04-25T00:00:00 +63022-07-16T12:59:37 0001-04-25T00:00:00 +63022-07-16T12:59:37 0001-04-25T00:00:00 +63022-07-16T12:59:37
 
+# verify timestamp data with formatting options
+query PPPPPP
+SELECT to_timestamp(null, '%+'), to_timestamp(0, '%s'), to_timestamp(1926632005, '%s'), to_timestamp(1, '%+', '%s'), to_timestamp(-1, '%c', '%+', '%s'), to_timestamp(0-1, '%c', '%+', '%s')
```

Comment:
I checked such tests in PG, they do not work there 🤔

Reply:
These queries are just mirrors of the existing tests that were there for to_timestamp without formatting options.